crates/cxx-qt-gen/src/generator/rust/threading.rs

```diff
@@ -63,12 +81,18 @@ pub fn generate(
         ],
         implementation: vec![
             quote! {
-                unsafe impl Send for #cxx_qt_thread_ident {}
-            },
-            quote! {
-                impl #cxx_qt_thread_ident {
-                    /// Queue the given closure onto the Qt event loop for this QObject
-                    pub fn queue<F>(&self, f: F) -> std::result::Result<(), cxx::Exception>
+                impl cxx_qt::Threading for #cpp_struct_ident {
+                    type BoxedQueuedFn = #cxx_qt_thread_queued_fn_ident;
+                    type ThreadingTypeId = cxx::type_id!(#cxx_qt_thread_ident_type_id_str);
+
+                    #[doc(hidden)]
```

Comment:
I don't think trait implementations need documentation :thinking: 
So we could remove `#[doc(hidden)]` here.
Should still leave it on `queue` and `threading_clone`, etc.